### PR TITLE
Add Issue Triaging document

### PR DESCRIPTION
**Pull request by trask**
This would serve as a reference for us, and also we could link to these definitions, e.g. when we tag something with `needs minimal repro`.

EDIT: we could rename `specification` label to `needs specification` to go with the `needs ...` naming scheme

### CODE REVIEW

File: ISSUES.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Ideally minimal repros should be posted to a github repository,
+and should include a `README.md` that explains exactly how to build and run the repro,
+and what the output of the repro is, and what you expect (or would like) the output of
+the repro to be.
+
+Note: issues with this label are automatically marked stale if there is no activity for 30 days,
+and then closed automatically if there is no response from the author within 30 days after that.
+
+## `needs external contribution`
```
Comment (trask) [inline ISSUES.md:25]: thinking to rename `contribution welcome` to this (I like the `needs ...` meaning it needs something to become unblocked)

File: ISSUES.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+This is something that fits the scope of the project,
+but that the maintainers aren't planning to take on themselves.
+
+Note: issues with this label are automatically closed if there is no activity for 365 days.
```
Comment (trask) [inline ISSUES.md:30]: I think this may be worth it to help control the issue backlog long-term

File: ISSUES.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Issue triaging
+
```
Comment (breedx-splk) [inline ISSUES.md:2]: It's not immediately apparent that the things being described below are labels.
```suggestion

The following commonly used labels are used to help triage issues.

```

File: ISSUES.md (hunk 1 above)
Comment (breedx-splk) [reply ISSUES.md:25]: I understand the desire for consistency, but this reads less community/collaboration and rather more like an us-or-them separation.

File: ISSUES.md (hunk 2 above)
Comment (mateuszrzeszutek) [reply ISSUES.md:30]: I like that idea 👍

**Review by mateuszrzeszutek — APPROVED** [review]

**Comment by trask** [comment]
@breedx-splk ptal, thx

**Review by breedx-splk — APPROVED** [review]
👍🏻

**Pull request merged**